mobile/src/main/java/rs/readahead/washington/mobile/views/adapters/GalleryRecycleViewAdapter.java

```diff
@@ -62,8 +60,9 @@ public GalleryRecycleViewAdapter(Context context, IGalleryMediaHandler galleryMe
     @NonNull
     @Override
     public ViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
-        View v = LayoutInflater.from(parent.getContext()).inflate(cardLayoutId, parent,false);
-        return new ViewHolder(v, this.selectable);
+        itemBinding = CardGalleryAttachmentMediaFileBinding.inflate(LayoutInflater.from(parent.getContext()), parent, false);
```

Comment:
This could be converted to local variable and initialize it  `        CardGalleryAttachmentMediaFileBinding itemBinding = CardGalleryAttachmentMediaFileBinding.inflate(LayoutInflater.from(parent.getContext()), parent, false);
`